masterlist.yaml

```diff
@@ -8654,16 +8760,12 @@ plugins:
         <<: *dirtyPlugin
         itm: 306
         udr: 10
-  - name: 'Requiem - Dragonborn.esp'
-    dirty:
-      - crc: 0x740bb9f0
-        <<: *dirtyPlugin
-        itm: 3
-  - name: 'Requiem - Hearthfires.esp'
-    dirty:
-      - crc: 0xa3a78f22
-        <<: *dirtyPlugin
-        itm: 1
+  # Must be loaded directly after Requiem.esp
+  - name: 'Requiem - Bugsmasher Edition.esp'
+    priority: -127
+  # Must be loaded directly after Requiem.esp
+  - name: 'Requiem - Legendary Bugsmasher Edition.esp'
```

Comment:
Technically no but it requires you to have both USLEEP and USKP active, that are of course incompatible (and already in the masterlist).